lib/matplotlib/patches.py

```diff
@@ -989,9 +990,8 @@ def set_xy(self, xy):
 
         Parameters
         ----------
-        xy : numpy array or iterable of pairs
-            The coordinates of the vertices as a Nx2
-            ndarray or iterable of pairs.
+        xy : Nx2 array-like
+            The coordinates of the vertices as a Nx2 array-like.
```

Comment:
could leave out "as a ..." (already given in the type).

Reply:
done